doc/classes/AudioEffectFilter.xml

```diff
@@ -14,22 +14,27 @@
 			Threshold frequency for the filter, in Hz.
 		</member>
 		<member name="db" type="int" setter="set_db" getter="get_db" enum="AudioEffectFilter.FilterDB" default="0">
+			Steepnes of the cutoff curve in dB per octave, also known as order of the filter. Higher orders have a more aggressive cutoff.
 		</member>
 		<member name="gain" type="float" setter="set_gain" getter="get_gain" default="1.0">
-			Gain amount of the frequencies after the filter.
+			Boosts frequencies at the cutoff.
```

Comment:
I find this to be worse. The subject is a floating-point property, but the description makes it sound like it's a method that does something when called.

Reply:
Hm, I see what you mean.

Reply:
Oh wait I see where I went wrong.

Reply:
```suggestion
			Gain amount of the frequencies after the filter.
```